playbooks/manage-jira/README.md

```diff
@@ -0,0 +1,115 @@
+# Jira Project Automation using Ansible
+This ansible role by default uses `simplified-scrum` template and `software` project type key  with a predefined set of  `Permission-Scheme`  to create a Jira project.
+
+The template and project type key can be changed by adding the following variables with their values
+
+`'template_key': '<value>'` and `'type_key': '<value>'` 
+in the `project` section of the inventory.
+eg.
+```yaml
+atlassian:
```

Comment:
I believe you need to add another level in this example - i.e.: `atlassian.jira.project`, right?